backend/server/repositories/userRepoMySql.js

```diff
@@ -28,7 +28,7 @@ const findByEmail = async (email) => {
     .catch(console.error);
 
   await query; // Wait for uid and isMaintainer to be updated
-  return { userId: _userId, isMaintainer: _isMaintainer };
+  return { _userId: _userId, _isMaintainer: _isMaintainer };
```

Comment:
At least for myself, I tend to use leading/trailing underscore for variable names for when the variable is (1) same name as method parameter (2) the variable is used internally only (i.e. not referenced outside of the method)